Tools/Catalog-Builder/templates/catalog_template.html

```diff
@@ -56,14 +56,21 @@ <h2>
                                 </h2>
                                 {{ item.project_one_line.value }}
                         </div>
-                        <div class="collapse" id="{{ make_id(item.name.value) }}-collapse" aria-labelledby="{{ make_id(item.name.value) }}-heading" data-parent="#accordionParent">
+                        <div class="collapse" id="{{ make_id(item.name.value) }}-collapse" aria-labelledby="" data-parent="#accordionParent">
                             <div class="container">
                                 <p>{{ item.project_overview.value }}</p>
                                 <h6>Maintainers</h6>
                                 {{ item.core_maintainers.value }}
                                 <p><a href="{{ repos[project] }}#readme">Code repository</a></p>
                                 {{ make_links(item) }}
                             </div>
+                            <script type="text/javascript">
+                                var url = document.location.toString();
+                                if ( url.match('#') ) {
+                                    var section = $('#'+url.split('#')[1]+'-collapse');
+                                    section.collapse('show');
+                                }
+                            </script>
```

Comment:
Bravo. I still can't wrap my head around what is happening here, but this looks good.

Reply:
Ha, thanks. This code basically looks for a "#" in the URL, and uses the text after the "#" to figure out which section to expand. Then it uses the bootstrap class `.collapse` with the method `.collapse('show')` to expand the section.